testcontainers/testcontainers-go
# fix: 1370 update to dario.cat/mergo as best as possible

fixes 1370 (I think)

## What does this PR do?

replaces mergo

## Why is it important?

so people can actually use the package

## Related issues

- fixes #1370

## How to test this PR

try to use the base package. If you don't get an error then it works.

the following are the files that still reference the old path, thanks to transitive dependencies. I somehow have a feeling you'll be waiting a bit on the compose-spec.use `go mod why github.com/imdario/mergo` from one of the related submodules to see why. Also, hoo boy, is that a lot of submodules you got there.

```
modules/compose/go.mod
33=require (
88:     github.com/imdario/mergo v0.3.15 // indirect

modules/compose/go.sum
403=github.com/ianlancetaylor/demangle v0.0.0-20200824232613-28f6c0f3b639/go.mod h1:aSSvb/t6k1mPoxDqO4vJh6VOCGPwU4O0C2/Eqndh1Sc=
404:github.com/imdario/mergo v0.3.5/go.mod h1:2EnlNZ0deacrJVfApfmtdGgDfMuh/nq6Ok1EcJh5FfA=
405:github.com/imdario/mergo v0.3.15 h1:M8XP7IuFNsqUx6VPK2P9OSmsYsI/YFaGil0uD21V3dM=
406:github.com/imdario/mergo v0.3.15/go.mod h1:WBLT9ZmE3lPoWsEzCh9LPo3TiwVN+ZKEjmz+hD27ysY=

modules/k3s/go.mod
15=require (
36:     github.com/imdario/mergo v0.3.15 // indirect

modules/k3s/go.sum
178=github.com/ianlancetaylor/demangle v0.0.0-20181102032728-5e5cf60278f6/go.mod h1:aSSvb/t6k1mPoxDqO4vJh6VOCGPwU4O0C2/Eqndh1Sc=
179:github.com/imdario/mergo v0.3.15 h1:M8XP7IuFNsqUx6VPK2P9OSmsYsI/YFaGil0uD21V3dM=
180:github.com/imdario/mergo v0.3.15/go.mod h1:WBLT9ZmE3lPoWsEzCh9LPo3TiwVN+ZKEjmz+hD27ysY=
```
